client/js/libs/handlebars/parse.js

```diff
@@ -108,20 +99,19 @@ function colors(line) {
     for (var i = 0; i < parseArr.length; i++) {
         text = parseArr[i];
         match = text.match(back_re);
-        colour = match && colourMap[+match[1]];
-        if (!match || !colour) {
+        if (!match) {
             // ^C (no colour) ending. Escape current colour and carry on
             background = "";
             continue;
         }
-        // set the background colour
-        // we don't overide the background local var to support nesting
-        if (colourMap[+match[3]]) {
-            background = " " + colourMap[+match[3]].back;
-        }
+		colour = "irc-fg" + +match[1];
+		// set the background colour
+		if (match[3]) {
+			background = " irc-bg" + +match[3];
+		}
```

Comment:
This whole file needs to be converted to tabs, and eslint ignores `handlerbars/` folder.